src/app/(verify)/verify/page.tsx

```diff
@@ -1,10 +1,27 @@
 "use client";
+import axios from "axios";
 import { useEffect, useState } from "react";
+import { toast } from "sonner";
 import { Button } from "~/components/ui/button";
 
 const Page = () => {
   const [countdown, setCountdown] = useState<number>(0);
 
+  const sendEmailVerification = async () => {
+    try {
+      const data = (await axios.post("/api/email-verification", {}))
+        .data as Record<string, string>;
```

Comment:
most of the time, there should never be something like `as Record<string, string>`, just type it more explicitly.

Edit: I realized you can probably move this into a tRPC route so it is type safe.

Reply:
Alright, I've moved this route to tRPC.